pkg/3scale/amp/component/apicast.go

```diff
@@ -13,6 +13,12 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	ApicastSecretRedisSecretName             = "apicast-redis"
+	ApicastSecretRedisProductionURLFieldName = "PRODUCTION_URL"
+	ApicastSecretRedisStagingURLFieldName    = "STAGING_URL"
```

Comment:
we could live without `Apicast` prefix, right? it is in `apicast.go` file, so context gives the remaining info. Specially because I see this constant is used just in this file

Reply:
same principle applies to other files

Reply:
does not apply because there could be name conflict as they all belong to the same package :+1: